deployer/scripts/util.sh

```diff
@@ -304,58 +304,6 @@ function wait_for_builds_complete()
     return 0
 }
 
-get_error_pod() {
-    # $1 is component for selector
-    oc get pods -l component=$1 | awk -v sel=$1 '$1 ~ sel && ($3 == "Error" || $3 == "CrashLoopBackOff") {print $1}'
-}
-
-wait_for_pod_ACTION() {
```

Comment:
Deleting this function appears to be a reason for [Origin Aggregated Logging Test](https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_aggregated_logging/99/console) failure, it is still used in `test-es-copy.sh`, `test-fluentd-forward.sh` and `test-viaq-data-model.sh`